release_notes/49-release-notes.adoc

```diff
@@ -0,0 +1,478 @@
+
+//External integrations
+
+//ROX-22434
+[id="servicenow-vm-integration_{context}"]
+=== Integration with the ServiceNow Container Vulnerability Response Application is GA
+
+{product-title-short} integration with the ServiceNow Container Vulnerability Response Application is now GA in the link:https://store.servicenow.com/store/app/edea7344476072502ec7c1c4f16d4343#summary[ServiceNow Marketplace]. {product-title-short} integration with ServiceNow populates rich container image vulnerability data from {product-title-short} in the ServiceNow Container Vulnerability Response Module. It enables {product-title-short} users to create custom vulnerability management workflows for efficient tracking and remediation of vulnerabilities.
+
+//ROX-30087
+[id="m2m-token-exchange-jwt"]
+=== Support for machine-to-machine (M2M) token exchange for external JSON web token (JWT) issuers
+
+With this update, {product-title-short} supports transparent M2M token exchange with external JWT token issuers. {product-title-short} performs a token exchange between third-party identity tokens and {product-title-short}. It uses role mapping to allow access to the {product-title-short} API. This enables third-party clients such as Prometheus server that do not support the full M2M token exchange flow to access the {product-title-short} API endpoint by using, for example, Kubernetes service account tokens.
+
+For more information, see xref:../configuration/configure-api-token.adoc#configure-api-token[Configuring API tokens].
+
+//ROX-29959, ROX-29070
+[id="declarative-config-m2m_{context}"]
+=== Declarative configuration for M2M access configuration
+
+You can now declaratively configure M2M OIDC authentication. To configure M2M authentication resources, you create YAML files that contain configuration information. These files are used to create a config map or secret that is added to Central by using a mount point during installation of the {product-title-short} Central resource. 
+
+For an example of setting up declarative configuration for short-lived OIDC token usage, see xref:../configuration/declarative-configuration-using.adoc#declarative-config-example-short-lived-token[Declarative configuration short-lived token example].
+
+//Security
+
+//ROX-20262
+[id="certificate-rotation-operator-managed-clusters_{context}"]
+=== Automatic certificate authority (CA) rotation for Operator-managed clusters
+
+With this update, the Central CA certificate, which has a validity of 5 years, will be automatically rotated for Operator-managed Central and secured clusters. Previously, in release 4.7, {product-title-short} automatically rotated the 1-year service certificates, and now rotation is provided for the CA certificates. This enhancement simplifies the management of large cluster installations, as manually upgrading certificates can be a significant task. 
```

Comment:
Suggestion:
Previously, in release 4.7, RHACS automatically rotated only the 1-year service certificates. This change enables automatic rotation for the 5-year CA certificates also. The change is designed to simplify the management of large cluster installations, where the manual upgrading of certificates can require significant effort.